theme/static/js/hs_file_browser.js

```diff
@@ -706,7 +707,9 @@ function bindFileBrowserItemEvents() {
                     }
                 }
                 $(this).toggle(agg_app || extension_app);
+                hasTools = hasTools || agg_app || extension_app;
             });
+            $("#tools-separator").toggleClass("hidden", !hasTools);
         }
```

Comment:
not sure if it applies would a ternary be easier to understand or just as complex https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
If this is in reference to the logical operation above, then no. A ternary operator is a shortcut for `if then else`